src/Akeneo/Apps/back/Application/Command/UpdateAppHandler.php

```diff
@@ -44,6 +46,8 @@ public function handle(UpdateAppCommand $command): void
 
         $app->setLabel(new AppLabel($command->label()));
         $app->setFlowType(new FlowType($command->flowType()));
+        $app->setImage(null !== $command->image() ? new AppImage($command->image()) : null);
```

Comment:
As we don't deal with image deletion, it would be better to call setImage only when you have an image. And that's more readable than having a ternary operator in a setter.
```suggestion
        if (null !== $command->image()) {
            $app->setImage(new AppImage($command->image()));
        }
```

Reply:
According to the discussion we had with Franck, that's ok like that :+1: